src/ngraph/pass/algebraic_simplification.cpp

```diff
@@ -614,15 +681,18 @@ static bool simplify_reduction(shared_ptr<Node> n)
 static unordered_map<NodeTypeInfo, function<bool(shared_ptr<Node>)>> initialize_ops_to_simplifiers()
 {
     return unordered_map<NodeTypeInfo, function<bool(shared_ptr<Node>)>>(
-        {{op::Add::type_info, simplify_add},
-         {op::Multiply::type_info, simplify_multiply},
-         {op::Concat::type_info, simplify_concat},
+        {{op::v0::Add::type_info, simplify_add},
+         {op::v0::Multiply::type_info, simplify_multiply},
          {op::v1::Gather::type_info, simplify_gather},
-         {op::Sum::type_info,
-          function<bool(shared_ptr<Node>)>{simplify_reduction<op::Sum, get_sum_constant>}},
-         {op::Product::type_info,
-          function<bool(shared_ptr<Node>)>{simplify_reduction<op::Product, get_prod_constant>}},
-         {op::Log::type_info, simplify_log}});
+         {op::v1::Add::type_info, simplify_add},
```

Comment:
We cant apply this transformation for v1::Add and v1::Multiply.
You need to update simplify_add/simplfy_mul to use remove_node_update_name otherwise it 100% will brake some existing models tests.

Reply:
sure, made these changes.